TodoApp/TodoApp.Api.Tests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.2.0" targetFramework="net472" />
```

Comment:
Remove these MS test packages (using NuGet package manager) and install NUnit framework instead. We use NUnit in KC and is better for continuous integration.
![image](https://user-images.githubusercontent.com/16819859/44954074-d61b1e00-ae9d-11e8-998f-1e9b3195d668.png)